java/generate/config.json

```diff
@@ -5,7 +5,7 @@
     "developerOrganizationUrl": "https://www.finbourne.com",
     "sourceFolder": "src/main/java",
     "groupId": "com.finbourne",
-    "artifactId": "lusid-sdk-preview",
+    "artifactId": "lusid-sdk",
```

Comment:
would it not be safer to leave this as preview and force the production generation to overwrite it?